docs/src/test-pom-js.md

```diff
@@ -9,65 +9,66 @@ We will create a `PlaywrightDevPage` helper class to encapsulate common operatio
 
 ```js js-flavor=js
 // playwright-dev-page.js
+const { expect } = require('@playwright/test');
+
 exports.PlaywrightDevPage = class PlaywrightDevPage {
-  /**
-   * @param {import('playwright').Page} page 
```

Comment:
We added this because a few users were unhappy and had no auto-completion. In general there was a trend to add JSDoc where it makes sense.